webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/servers/NewNodeManager.scala

```diff
@@ -660,19 +585,18 @@ trait ComposedNewNodeManager extends NewNodeManager with NewNodeManagerHooks {
 
          // Update hooks for the node
          afterNodeAcceptedAsync(id)
-
          acceptationResults
        }
     )
 
     // If one node succeed, then update policies
     if ( acceptanceResults.exists{ case Full(_) => true; case _ => false } ) {
       updateDynamicGroups.startManualUpdate
+      cacheToClear.foreach { _.clearCache() }
```

Comment:
why do you need to clear cache? updateDynamicGroup will trigger a policy generation by itself

Reply:
Not sur. It was done like that in the UI accept. I think it's bubious (we should never have to clear cache, it causes massive perf penalty). I'm going to remove it.